x-pack/plugins/security_solution/server/lib/entity_analytics/asset_criticality/constants.ts

```diff
@@ -33,3 +33,13 @@ export const assetCriticalityFieldMap: FieldMap = {
     required: false,
   },
 } as const;
+
+/**
+ * CriticalityModifiers are used to adjust the risk score based on the criticality of the asset.
+ */
+export const CriticalityModifiers = {
```

Comment:
Would this be overkill?

```
export const CriticalityModifiers: Record<AssetCriticalityRecord['criticality_level'], number> = {

```

Reply:
Not at all: 01511891d24ba7466f263eb7574d19633cfb577b